<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLiteCreatorDialogForm</class>
 <widget class="QDialog" name="SQLiteCreatorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>234</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SQLite Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="m_datasourceTypeTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SQLite</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>103</height>
            </size>
           </property>
           <property name="title">
            <string>Creation Information</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="m_fileLabel_2">
                  <property name="text">
                   <string>New Database</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
                  <item>
                   <widget class="QLineEdit" name="m_fileLineEdit"/>
                  </item>
                  <item>
                   <widget class="QToolButton" name="m_searchDatabaseToolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="m_allowIntermediateDirCheckBox">
                  <property name="text">
                   <string>Allow intermediate directory creation</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_advancedConnectionOptionsCheckBox">
           <property name="text">
            <string>Advanced Connection Options</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="m_advancedOptionsGroupBox">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>134</height>
          </size>
         </property>
         <property name="title">
          <string>Advanced Options</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>-64</y>
               <width>201</width>
               <height>194</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Wich Spatial Metatables</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QRadioButton" name="m_spatialiteSchemaRadioButton">
                     <property name="text">
                      <string>SpatiaLite</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="m_ogrSchemaRadioButton">
                     <property name="text">
                      <string>OGR/FDO</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QCheckBox" name="m_createTeRasterMetadataCheckBox">
                  <property name="text">
                   <string>Create raster metadata tables</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="m_hideSpatialMetatablesCheckBox">
                  <property name="text">
                   <string>Hide spatial metadata tables</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="m_hideRasterTablesCheckBox">
                  <property name="text">
                   <string>Hide raster tables</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QLabel" name="m_tablesToHideLabel">
                  <property name="text">
                   <string>Tables to Hide</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="m_tablesToHideLineEdit"/>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>122</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
       <item>
        <widget class="QPushButton" name="m_helpPushButton">
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="m_applyPushButton">
           <property name="text">
            <string>&amp;Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_closePushButton">
           <property name="text">
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_fileLineEdit</tabstop>
  <tabstop>m_searchDatabaseToolButton</tabstop>
  <tabstop>m_allowIntermediateDirCheckBox</tabstop>
  <tabstop>m_advancedOptionsGroupBox</tabstop>
  <tabstop>m_spatialiteSchemaRadioButton</tabstop>
  <tabstop>m_ogrSchemaRadioButton</tabstop>
  <tabstop>m_createTeRasterMetadataCheckBox</tabstop>
  <tabstop>m_hideSpatialMetatablesCheckBox</tabstop>
  <tabstop>m_hideRasterTablesCheckBox</tabstop>
  <tabstop>m_tablesToHideLineEdit</tabstop>
  <tabstop>m_applyPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_closePushButton</sender>
   <signal>pressed()</signal>
   <receiver>SQLiteCreatorDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_advancedConnectionOptionsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_advancedOptionsGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
